android/javatests/org/chromium/chrome/browser/privacy/settings/BravePrivacySettingsTest.java

```diff
@@ -41,8 +41,17 @@
     private static final String PREF_CLEAR_BROWSING_DATA = "clear_browsing_data";
     private static final String PREF_PRIVACY_SANDBOX = "privacy_sandbox";
 
-    // Ignore "usage_stats_reporting" and "privacy_sandbox"
-    private static int PRIVACY_SETTINGS_NUMBER_OF_ITEMS = 7;
+    // Ignore usage_stats_reporting and privacy_sandbox
+    // Items counted:
+    // - clear_browsing_data
+    // - safe_browsing
+    // - https_first_mode
+    // - can_make_payment
+    // - preload_pages
+    // - secure_dns
+    // - do_not_track
+    // - sync_and_services_link
+    private static int PRIVACY_SETTINGS_NUMBER_OF_ITEMS = 8;
```

Comment:
this seems fragile, is there another way we can get this count? Like the `original_value - things we take out`

Reply:
cc: @samartnik

Reply:
Addressed here https://github.com/brave/brave-core/commit/ef1bffcb3b9898c07ec1956a4b4742807a6c9df2